### PR TITLE
Add python-potr (pure Python Off-The-Record encryption) version 1.0.1

### PR DESCRIPTION
**Pull request by np**

### CODE REVIEW

**Comment by np** [comment]
The error message is unrelated to this package and building it worked fine on my machine.

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -16154,5 +16154,26 @@ let
...
+      platforms   = platforms.all;
+      maintainers = with maintainers; [ np ];
+    };
+
+    propagatedBuildInputs = [ self.pycrypto ];
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/p/python-potr/python-${name}.zip";
+      sha256 = "a8a23fc1aac1e9c97b02fe462097f54f2d247f7979f0aa8ce4d66f88f6927bac";
+      md5 = "cb7c7d812ca548e81175e9e0298d70ac";
```
Comment (domenkozar) [inline pkgs/top-level/python-packages.nix]: please remove `md5` line

**Comment by globin** [comment]
I'm pretty sure I've added this in the meantime for some poezio plugin dependency

**Comment by domenkozar** [comment]
Indeed, potr is already in our repository. Thanks for the effort!

**Pull request closed without merging**